script/dev-setup

```diff
@@ -31,10 +31,9 @@ install-asdf-plugins
 info "Installing bundler (if necessary)"
 gem install bundler --conservative --version "$BUNDLER_VERSION"
 
-info "Installing postgres (if necessary)"
-if ! cmd psql; then
-  brew-install postgresql
-fi
+info "Installing postgres + postgis (if necessary)"
+brew-install postgresql
```

Comment:
Feedback: because this can trigger a long Homebrew update, maybe we should keep the `cmd psql` check from before and see if we can add a check for `postgis` (it might have an associated binary yeah?)

Reply:
I just ran `brew install postgresql` and it took a minute to execute, output a bunch of text, and then emitted warning: `Warning: postgresql 12.3_4 is already installed and up-to-date`

Reply:
That's interesting. It's a lot faster on my machine. But no worries, a good suggestion.

I put in `if ! cmd` blocks again. Postgis doesn't seem to have a conveniently named executable, but I do know for sure it installs the `shp2pgsql` shape file converter, so I set it up to look for that.

Reply:
RE: the time it took, it took me longer because brew had some updates that it could install, so that'll be variable. 

I see these executables are available. Yeah, no super clear "postgis":
```bash
❯ brew list postgis
/usr/local/Cellar/postgis/3.0.1_3/bin/create_undef.pl
/usr/local/Cellar/postgis/3.0.1_3/bin/pgsql2shp
/usr/local/Cellar/postgis/3.0.1_3/bin/postgis_proc_upgrade.pl
/usr/local/Cellar/postgis/3.0.1_3/bin/postgis_restore.pl
/usr/local/Cellar/postgis/3.0.1_3/bin/profile_intersects.pl
/usr/local/Cellar/postgis/3.0.1_3/bin/raster2pgsql
/usr/local/Cellar/postgis/3.0.1_3/bin/shp2pgsql
/usr/local/Cellar/postgis/3.0.1_3/bin/test_estimation.pl
/usr/local/Cellar/postgis/3.0.1_3/bin/test_geography_estimation.pl
/usr/local/Cellar/postgis/3.0.1_3/bin/test_geography_joinestimation.pl
/usr/local/Cellar/postgis/3.0.1_3/bin/test_joinestimation.pl
```